pkg/arvo/gen/who.hoon

```diff
@@ -0,0 +1,28 @@
+/+  *server
+::  @p request handler
+::
+|=  [[now=@da eny=@ bek=beak] ~ ~]
+|=  [authorized=? =request:http]
+^-  simple-payload:http
+=/  url=(unit pork:eyre)
+  (rush url.request apat:de-purl:html)
+::
+::  url doesn't match expected binding from :launch
+::
+?.  ?=(^ url)
+  ~|  [%invalid-url url.request]  !!
+?.  ?=([%who *] q.u.url)
+  ~|  [%unknown-url url.request]  !!
+::
+::
+?.  ?=(~ t.q.u.url)
+  [[%404 ~] ~]
+=/  response=json
+  (frond:enjs:format %who (ship:enjs:format p.bek))
+:-
+  :-  %200
+  :~
+    ['content-type' 'application/json']
+    ['access-control-allow-origin' 'https://bridge.urbit.org']
```

Comment:
Here and in launch's `+on-init`:

```suggestion
  :~  ['content-type' 'application/json']
      ['access-control-allow-origin' 'https://bridge.urbit.org']
```

Opening this to just Bridge is fine for the time being. I need to write that CORS handling proposal...